$MirOS: ports/www/lynx/patches/patch-samples_lynxdump,v 1.2 2008/12/17 00:24:42 tg Exp $

	site policy

--- samples/lynxdump.orig	Sat Oct  6 22:02:06 2001
+++ samples/lynxdump	Sat Mar  3 18:20:22 2012
@@ -7,9 +7,10 @@ umask 077
 MYTMP=$TMPDIR/mytmp$$
 mkdir $MYTMP || exit 1
 trap 'cd /; rm -rf $MYTMP' 0 1 2 5 15
-if test $HOME/.lynxrc ; then
-  cp $HOME/.lynxrc $MYTMP/.lynxrc
+mkdir $MYTMP/.etc || exit 1
+if test -f $HOME/.etc/lynxrc ; then
+  cp $HOME/.etc/lynxrc $MYTMP/.etc/lynxrc
 fi
-echo 'keypad_mode=NUMBERS_AS_ARROWS' >> $MYTMP/.lynxrc
+echo 'keypad_mode=NUMBERS_AS_ARROWS' >>$MYTMP/.etc/lynxrc
 HOME=$MYTMP; export HOME
 lynx -justify -dump -force_html -with_backspaces -nolist $*
